org.eclipse.xtext.common.types.tests/tests/org/eclipse/xtext/common/types/access/impl/AbstractTypeProviderTest.java

```diff
@@ -3890,4 +3866,34 @@ public void testBug470767() {
 		JvmTypeReference paramType = method.getParameters().get(0).getParameterType();
 		assertEquals("int", paramType.getSimpleName());
 	}
+
+	@Test
+	public void testInnerAnnotationType() throws Exception {
```

Comment:
What is the cause for the change to this test?

Reply:
asm fixed a bug/bad behaviour. thus the change/additional test cases + the fix

Reply:
yes